pyproject.toml

```
@@ -37,7 +37,7 @@ dependencies = [
 [project.optional-dependencies]
 examples = [
     "jupyter ~=1.0",
-    "otelib ~=0.3.2",
+    "otelib ~=0.4.1",
```

```suggestion
    "otelib ~=0.3.2",
```
Do not yet support pydantic v2.